frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -17,18 +17,20 @@ export interface NodeInfo {
 }
 
 export default class NodesApi extends ExplorerApi {
-  async getNodes(limit: number = 15) {
+  async getNodes(limit: number = 15, endTimestamp?: number) {
```

Comment:
I still encourage you to think about the argument naming from the usage perspective. `endTimestamp` does not have a clear meaning when you query a list of nodes.